analysis/topology/topological_space.lean

```diff
@@ -775,6 +775,35 @@ lemma is_closed_induced_iff [t : topological_space β] {s : set α} {f : α →
 ⟨assume ⟨t, ht, heq⟩, ⟨-t, by simp; assumption, by simp [preimage_compl, heq.symm]⟩,
   assume ⟨t, ht, heq⟩, ⟨-t, ht, by simp [preimage_compl, heq.symm]⟩⟩
 
+lemma induced_comp {α β γ : Type*} (f : α → β) (g : β → γ) :
```

Comment:
Isn't this already `induced_compose`, up to minor details?

Reply:
I guess it is. And in fact, I don't need this lemma anymore. So I will remove it.